manuals/manual_convert_schedule_to_events.md

```diff
@@ -0,0 +1,463 @@
+|--------------------------|--------------------------------|------------------------------------------------------------------------------|
+| ETA Port                 | port.eta.carrier               | The estimated time the ship will arrive at the port.                         |
+| ATA Port                 | port.ata.carrier               | The actual time the ship has arrived in the port.                            |
+| ETA Pilot Boarding Place | pilotBoardingPlace.eta.carrier | The estimated time the ship has arrived at the pilot boarding place          |
+| ATA Pilot Boarding Place | pilotBoardingPlace.ata.carrier | The actual time the ship will arrive at the pilot boarding place             |
+| ETA Berth                | berth.eta.carrier              | The estimated time the ship will arrive at the berth (before it's moored)    |
+| ATA Berth                | berth.ata.carrier              | The actual time the ship has arrived at the berth                            |
+| ETD Berth                | berth.etd.carrier              | The estimated time the ship will depart from the berth (after it's unmoored) |
+| ATD Berth                | berth.atd.carrier              | The actual time the ship has departed from the berth                         |
+| ETD Pilot Boarding Place | pilotBoardingPlace.etd.carrier | The estimated time the ship has depart through the pilot boarding place      |
+| ATD Pilot Boarding Place | pilotBoardingPlace.atd.carrier | The actual time the ship will depart through pilot boarding place            |
+| ETD Port                 | port.etd.carrier               | The estimated time the ship will depart from the port                        |
+| ATD Port                 | port.atd.carrier               | The actual time the ship has departed from the port                          |
+| Cancel berth visit       | berth.cancel.carrier           | A previously planned visit to a specific berth will no longer happen         |
+| Cancel visit             | port.cancel.carrier            | A previously planned visit to the port will no longer happen                 |
+
+### Port Call ID
+
+The port call id is a **unique, non-repeating** identifier which the system who reads your events can use to group them together. This is nessasary to work with estimations, if for example you send an ETA Port for a vessel and later send another ETA Port which is 2 days apart it could mean either
+
+1. The ETA Port for the visit has changed
+2. The vessel will make a new visit within two days of it's previous visit
+
+By supplying a port call ID consumers can differentiate between the two: if the id is identical the event is an update, if the id is different it is a second visit.
+
+A generated port call id will have the following structure `PID-` followed by a string uniquely identify your system, e.g. your source string. 
+It is then followed by a unique identifier of your choosing.
+
+### Berth visit ID
```

Comment:
Is this section relevant for this manual?

Reply:
I am not sure if they are able to report multiple berths (Robbert suggested that could) but if they do it would be wise if they'd be able to use something which allows us to identify what belongs together.